common/models/transducer/alignment.py

```diff
@@ -1,16 +1,19 @@
 from returnn.tf.util.basic import get_shape_dim, check_input_dim
+from returnn.tf.util.data import Data
 
 
 def rna_alignment(source, **kwargs):
+  """ Used only to create alignments according to RNA loss function.
+  B: batch, T: time, U:target/labels, V: vocabulary
+  Args:
+      source: function (i: int, as_data: bool = False, ...) -> tf.Tensor|Data, ...)
```

Comment:
This is still wrong. Please look carefully.

```suggestion
      source: function (i: int, as_data: bool = False, ...) -> tf.Tensor|Data
```
(As a generic rule, brackets should always be balanced.)